test/statetest/statetest_runner.cpp

```diff
@@ -9,7 +9,10 @@
 
 namespace evmone::test
 {
-void run_state_test(const StateTransitionTest& test, evmc::VM& vm)
+/// Execute the state @p test using the @p vm.
+///
+/// @param trace  Output execution summary to the default trace stream.
+void run_state_test(const StateTransitionTest& test, evmc::VM& vm, bool trace)
```

Comment:
this would be much less confusing
```suggestion
void run_state_test(const StateTransitionTest& test, evmc::VM& vm, bool trace_summary)
```

Reply:
Done.